src/class-parsely.php

```diff
@@ -937,10 +937,12 @@ public function get_clean_parsely_page_value( string $val ): string {
 	/**
 	 * Get the URL of the plugin settings page.
 	 *
+	 * @param int $_blog_id The Blog ID for the multisite subsite to use for context (Default null for current).
+	 *
 	 * @return string
 	 */
-	public static function get_settings_url(): string {
-		return admin_url( 'options-general.php?page=' . self::MENU_SLUG );
+	public static function get_settings_url( int $_blog_id = null ): string {
+		return get_admin_url( $_blog_id, 'options-general.php?page=' . self::MENU_SLUG );
```

Comment:
Have we got any multisite tests for this that could be updated?

Reply:
we do now.